Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a group of VMs/Performance Analysis for a group of VMs.workbook

```diff
@@ -15,7 +15,7 @@
         "version": "KqlParameterItem/1.0",
         "query": "",
         "crossComponentResources": [
-          "{Workspaces}"
+          "value::all"
```

Comment:
are you sure you want all?  do you want "any one" or whatever?

Reply:
Thank you. I had set it as 'Any One' only, not sure why it picked up 'All'. 
I went back to setting for the param and reset it - apparently it removed everything inside "crossComponentResources".

I think this should work.